docs/CommunicationIOS.md

```diff
@@ -4,7 +4,7 @@ title: Communication between native and React Native
 layout: docs
 category: Guides (iOS)
 permalink: docs/communication-ios.html
-next: native-modules-android
+next: packaging-a-RNComponent-ios
```

Comment:
We use lower case for these links.
Also RN is not needed because everything here is RN.
How about `packaging-ios-component`?